controllers/user.controller.js

```diff
@@ -0,0 +1,48 @@
+const User = require("../models/user.model");
+
+exports.create = function(req, res){
+    let user = new User(
```

Comment:
```const user``` would be better here because user variable won't be changed

Reply:
cool